src/main/java/hudson/plugins/jacoco/portlet/bean/JacocoDeltaCoverageResultSummary.java

```diff
@@ -0,0 +1,130 @@
+package hudson.plugins.jacoco.portlet.bean;
+
+import hudson.model.Job;
+import hudson.model.Run;
+import hudson.plugins.jacoco.JacocoPublisher;
+import hudson.plugins.jacoco.portlet.JacocoLoadData;
+
+import java.math.BigDecimal;
+
+/**
+ * This class encapsulates actual delta coverage of current build.
+ * It calculates absolute difference between coverage of last successful build and current build
+ */
+public class JacocoDeltaCoverageResultSummary {
+
+    /**
+     * Variables to capture delta coverage of current build
+     */
+    private BigDecimal instructionCoverage;
+
+    private BigDecimal branchCoverage;
+
+    private BigDecimal complexityCoverage;
+
+    private BigDecimal lineCoverage;
+
+    private BigDecimal methodCoverage;
+
+    private BigDecimal classCoverage;
+
+    private boolean coverageBetterThanPrevious=false;
+
+    public JacocoDeltaCoverageResultSummary() {
+    }
+
+    // Used to extract coverage result of current and last successful build and encapsulate delta coverage values
+    public static JacocoDeltaCoverageResultSummary build(Run<?,?> run){
+        Job<?, ?> parent = run.getParent();
+        Run<?,?> lastSuccessfulBuild = parent!=null ? parent.getLastSuccessfulBuild():null;
+        JacocoCoverageResultSummary lastBuildCoverage = lastSuccessfulBuild!=null ? JacocoLoadData.getResult(lastSuccessfulBuild):new JacocoCoverageResultSummary();
+        JacocoCoverageResultSummary currentBuildCoverage = JacocoLoadData.getResult(run);
+
+        JacocoDeltaCoverageResultSummary jacocoDeltaCoverageResultSummary = new JacocoDeltaCoverageResultSummary();
+        jacocoDeltaCoverageResultSummary.instructionCoverage = lastBuildCoverage.getInstructionCoverage().subtract(currentBuildCoverage.getInstructionCoverage()).abs();
```

Comment:
This looks strange to me, why don't we keep the negative values so we know the actual difference?